Automattic/jetpack
# Boost: Update performance improvement UI in dashboard.

Fixes #20680

#### Changes proposed in this Pull Request:
The speed score currently displays improvements(if any) between the latest score and the previous score measured in boost. This PR changes that and compares the latest score with the state of all boost optimizations being disabled.

#### Jetpack product discussion
pbIAo1-hO-p2#comment-784

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Go to boost options page.
* *Refresh* the score and see if the _previous score_ resembles the score of boost being inactive.
